SpotifyWebApi/Model/LinkedFrom.cs

```diff
@@ -38,5 +38,11 @@ public class LinkedFrom
         /// </summary>
         [JsonProperty("uri")]
         public SpotifyUri Uri { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total count.
+        /// </summary>
+        [JsonProperty("total")]
+        public int Total { get; set; }
```

Comment:
Seems like this is not needed here.

Reply:
The one you linked: https://developer.spotify.com/documentation/web-api/reference/#/operations/get-playlist, you're actually using FullPlaylist, which makes sense because it has more data and spotify doesn't return all the tracks and has to be paged but for the ones that are using SimplePlaylist. I went through the calls that use that one.

https://developer.spotify.com/documentation/web-api/reference/#/operations/get-list-users-playlists
https://developer.spotify.com/documentation/web-api/reference/#/operations/get-a-list-of-current-users-playlists
https://developer.spotify.com/documentation/web-api/reference/#/operations/search
https://developer.spotify.com/documentation/web-api/reference/#/operations/get-featured-playlists

In each of them, what comes back is a list of playlists with Tracks that only has 2 properties: Href and Total
SimplePlaylist is just a summary of what the playlist has.

```
"items": [
   "collaborative": boolean,
   "description": string,
   ... other properties
   "tracks": {
      "href": string,
      "total": number
   }
]
```

I used the console tool to see the result from Spotify and try any of the calls in the Api that used SimplePlaylist to also see the json coming back https://developer.spotify.com/console/get-current-user-playlists/

Example item tracks result for calls that use SimplePlaylist if you try the above calls in their console.
```
"tracks": {
   "href": "https://api.spotify.com/v1/playlists/4YLyYF8RO666VM7EyrBm8v/tracks",
   "total": 57
}
```

Reply:
It seems you are correct, did not double-check the link I send you.

Although now I'm debating if this should be in the `LinkedFrom` class. If I look at the OpenAPI spec (see: https://community.spotify.com/t5/Spotify-for-Developers/OpenApi-Swagger-description-for-the-Web-API/td-p/5196705 and https://github.com/sonallux/spotify-web-api) they use a different type for reference to the tracks `PlaylistTracksRef`

I would say we create a new class specifically for this purpose:
```csharp
public class PlaylistTracksRef
{
    /// <summary>
    /// A link to the Web API endpoint where full details of the playlist's tracks can be retrieved.
    /// </summary>
    [JsonProperty("href")]
    public string Href { get; set; }

    /// <summary>
    /// Number of tracks in the playlist.
    /// </summary>
    [JsonProperty("total")]
    public int? Total { get; set; }
}
```

What are your thoughts on this?

Reply:
Yes, I think that makes the most sense.